pkg/client/client.go

```diff
@@ -122,6 +124,20 @@ func NewClient(cfg *config.DataSetConfig, client *http.Client, logger *zap.Logge
 		addEventsEndpointUrl += "/api/addEvents"
 	}
 
+	userAgent := fmt.Sprintf(
+		"%s;%s;%s;%s;%s;%s;%d",
+		"datasetexporter",
```

Comment:
We should change this to dataset-go

Reply:
good idea, I have inherited the name from previous Martins PR https://github.com/scalyr/dataset-go/pull/25